### PR TITLE
UG changes for processing

### PR DESCRIPTION
**Pull request by SimNee**
fixes issues #208 #209 #192

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -486,63 +486,138 @@ You can then run `sort rate` to obtain the list of donors with higher success ra
...
+* `Not Processing` : The current donor or patient is not yet matched with anyone and is not going for any cross-matching test yet.
+
+* `Processing` : The current donor or patient found a match already and is in the midst of preparing for a cross-matching test.
+
+* `Done` : The current donor or patient have completed the cross-matching and the result of the cross-matching shows a positive result.
+Hence, this donor or patient do not need to be in ORGANice anymore.
+
+All the patients and donors should be `Not Processing` initially.
+When the command is used for the first time for the patient and donor pair, the `Status` of the patient and donor will change from `Not Processing` to `Processing`.
+If the patient and donor is being processed, their `Status` will remain `Processing`.
```
Comment (Yen-Peng) [inline docs/UserGuide.adoc]: remain as

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -486,63 +486,138 @@ You can then run `sort rate` to obtain the list of donors with higher success ra
...
+
 === Processing potential matches: `processing`
-//TODO: change to shorter term that captures the meaning
 
-After finding a list of matches, the hospital is free to schedule a cross-matching test between
-a specific donor and patient. +
+After finding a list of matches, the hospital will be able to schedule a cross-matching test between
+a specific donor and patient.
+To schedule a cross-matching test, there will be a lot of other administrative tasks to be done.
+Hence, this command will be able to allow you to have an overview on what kind of tasks you need to do for that
```
Comment (Yen-Peng) [inline docs/UserGuide.adoc]: overview of

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -486,63 +486,138 @@ You can then run `sort rate` to obtain the list of donors with higher success ra
...
-update the results in ORGANice. Cross matching tests have two possible results: pass or fail.
+update the results in ORGANice.
+Cross matching tests have two possible results: pass or fail.
 
-Format: `done icP/**NRIC OF PATIENT** icD/**NRIC OF DONOR** res/[pass/fail]`
-
-`icP/**NRIC OF PATIENT**` refers to the NRIC of the patient, while `icD/**NRIC OF DONOR**`
-refers to the NRIC of the donor.
+Format: `done ic/**PATIENT NRIC** ic/**DONOR NRIC** res/[pass/fail]`
+The order of the NRICs does not matter, as long as the patient and the donor must be valid.
```
Comment (Yen-Peng) [inline docs/UserGuide.adoc]: suggested addition: must **both** be valid

File: docs/UserGuide.adoc [hunk 4]
```diff
@@ -486,63 +486,138 @@ You can then run `sort rate` to obtain the list of donors with higher success ra
...
 If the result of the cross matching is a 'fail': +
 
 . Run the command based on the given format
 
-. The patient and donor will be added back to the matching pool for
+. The donor and patient will be added back to the matching pool for
 match detection with other patients and donors.
 
-. The patient-donor pair will not be considered a potential match in future match detection.
+. The patient and donor statuses will be marked to `Not Processing`.
```
Comment (Yen-Peng) [inline docs/UserGuide.adoc]: marked as

File: docs/UserGuide.adoc [hunk 5]
```diff
@@ -486,63 +486,138 @@ You can then run `sort rate` to obtain the list of donors with higher success ra
...
+
+. The donor and patient pair will not be considered a potential match in future match detection.
```
Comment (Yen-Peng) [inline docs/UserGuide.adoc]: suggested addition: potential match **anymore** in future match **detections**

File: docs/UserGuide.adoc (hunk 1 above)
Comment (SimNee) [reply docs/UserGuide.adoc]: amended

File: docs/UserGuide.adoc (hunk 2 above)
Comment (SimNee) [reply docs/UserGuide.adoc]: amended

File: docs/UserGuide.adoc (hunk 3 above)
Comment (SimNee) [reply docs/UserGuide.adoc]: amended

File: docs/UserGuide.adoc (hunk 4 above)
Comment (SimNee) [reply docs/UserGuide.adoc]: amended

File: docs/UserGuide.adoc (hunk 5 above)
Comment (SimNee) [reply docs/UserGuide.adoc]: amended

**Pull request merged**